src/reporters/http_sender.js

```diff
@@ -63,7 +63,7 @@ export default class HTTPSender {
       protocol: this._url.protocol,
       hostname: this._url.hostname,
       port: this._url.port,
-      path: this._url.pathname,
+      path: this._url.path,
```

Comment:
I am not seeing `path` being a valid property of the URL object: https://nodejs.org/api/url.html#url_class_url

Reply:
https://nodejs.org/api/url.html#url_urlobject_path